### ISSUE TITLE
iOS Failed to compile (v 1.1)

### ISSUE THREAD

**Issue by staticdreams** [NON906/HandMR#5]
Hello,

I'm trying to get the projet to work on iPhone.
I'm also using AR Foundation Remote 2.0

In both cases I'm getting an error in Xcode when compiling the application:

```Undefined symbols for architecture arm64:
  "_multiHandGetGesture", referenced from:
      _HandVRMain_multiHandGetGesture_m6D95AD5C22BB59AE8FFD6A009FCD17F557AA23E2 in Assembly-CSharp.o
      _HandVRMain_GetGestures_m5087DB6F80F3EBB417108461F5A4B30E773F530D in Assembly-CSharp.o
     (maybe you meant: _HandVRMain_multiHandGetGesture_m6D95AD5C22BB59AE8FFD6A009FCD17F557AA23E2)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I tried compiling for iOS 13 et iOS 15. No luck. Any advice?

UPDATE: I'm trying to compile for AR, maybe that is the problem? In this case how to I switch from VR to AR?

**Comment by NON906** [comment]
The URL of the iOS plugin was incorrect.
I fixed it (0d0af69c5e222e17c512bc5c4de98817437955e3), so please update and execute "Step 5" again.

**Comment by staticdreams** [comment]
Thanks! 
Work better now!
